tools/preflight/helper.go

```diff
@@ -201,6 +219,87 @@ func getVersionsOfGroup(grp string) ([]string, error) {
 	return apiVerList, nil
 }
 
+//  checkAndCreateVolumeSnapshotCRDs checks and creates volumesnapshot and related CRDs if not present on cluster.
+func checkAndCreateVolumeSnapshotCRDs(ctx context.Context) error {
```

Comment:
This func seems to perform a check which can be considered as one of the steps of preflight checking.
Making this func as part of `Run` struct seems more appropriate. 

This is not a hard requirement. Reconsider the above comment and make changes if necessary.

Reply:
there was an already similar function `checkCSI` which did the exact same thing except for the creation part which I missed earlier. the function is now moved to be a receiver function of Run type as it should be.